pkg/errors/errorMessage.go

```diff
@@ -146,8 +146,15 @@ var (
 	GALASA_ERROR_RETRIEVING_ARTIFACTS_FAILED    = NewMessageType("GAL1073E: Could not get run artifacts. Reason: '%s'", 1073, STACK_TRACE_WANTED)
 	GALASA_ERROR_DOWNLOADING_ARTIFACT_FAILED    = NewMessageType("GAL1074E: Could not download artifact '%s'. Reason: '%s'", 1074, STACK_TRACE_WANTED)
 
-	GALASA_ERROR_INVALID_RUN_NAME = NewMessageType("GAL1075E: run name '%s' is invalid. Expected it to be in format starting with letters, and ending in a number with no non-alphanumeric characters.", 1075, STACK_TRACE_NOT_WANTED)
+	GALASA_ERROR_INVALID_RUN_NAME               = NewMessageType("GAL1075E: run name '%s' is invalid. Expected it to be in format starting with letters, and ending in a number with no non-alphanumeric characters.", 1075, STACK_TRACE_NOT_WANTED)
+
+	GALASA_ERROR_INVALID_FROM_OR_TO_PARAMETER   = NewMessageType("GAL1076E: Badly formed from or to value '%s' specified in the age parameter. The value could not be converted into an integer value.", 1076, STACK_TRACE_NOT_WANTED)
+	GALASA_ERROR_FROM_AGE_SMALLER_THAN_TO_AGE   = NewMessageType("GAL1077E: Invalid value '%s' detected for age parameter. The from value must be older than the to value.", 1077, STACK_TRACE_NOT_WANTED)
+	GALASA_ERROR_FROM_AGE_NOT_SPECIFIED         = NewMessageType("GAL1078E: No from value has been specified or a value of 0 has been given in '%s'. A from value must be specified if using the age parameter.", 1078, STACK_TRACE_NOT_WANTED)
+	GALASA_ERROR_INVALID_AGE_PARAMETER          = NewMessageType("GAL1079E: Badly formed age parameter '%s' specified. Age of the tests should be specified in FROM:TO or FROM format with the units w (weeks), d (days) or h (hours).", 1078, STACK_TRACE_NOT_WANTED)
+
+	GALASA_ERROR_NO_RUNNAME_OR_AGE_SPECIFIED    = NewMessageType("GAL1080E: Neither a runname or an age have been specified. Either specify a runname or an age of the tests you want to see.", 1080, STACK_TRACE_NOT_WANTED)
```

Comment:
Neither ... nor is harder to read than Either ... or - best to avoid it if possible.